package.yaml

```diff
@@ -85,6 +85,8 @@ library:
   - unordered-containers
   - retry
   - github
+  - transformers
+  - lens-family-core
```

Comment:
```suggestion
```

We are already importing `lens-family-core` in line 65

Reply:
Oops. I might alphabetise this list to make it a bit easier to scan...